packages/backend-defaults/src/entrypoints/actionsRegistry/DefaultActionsRegistryService.ts

```diff
@@ -67,13 +67,19 @@ export class DefaultActionsRegistryService implements ActionsRegistryService {
         actions: Array.from(this.actions.entries()).map(([id, action]) => ({
           id,
           ...action,
+          attributes: {
+            // todo(blam): what's safe defaults?
+            destructive: action.attributes?.destructive ?? false,
```

Comment:
Wonder if we should set this to `true` a la `modelcontextprotocol/sdk` by default: https://github.com/modelcontextprotocol/typescript-sdk/blob/dd69efa1de8646bb6b195ff8d5f52e13739f4550/src/types.ts#L777-L812